libraries/standard/mqtt/src/iot_mqtt_network.c

```diff
@@ -105,6 +105,90 @@ static void _flushPacket( void * pNetworkConnection,
                           const _mqttConnection_t * pMqttConnection,
                           size_t length );
 
+/* Utility types for MQTT serializer override functions */
+typedef uint8_t ( * _getPacketTypeFunc_t )( void *,
```

Comment:
These typedefs can also be moved to ```iot_mqtt_types.h``` and can be used by ```IotMqttSerializer_t``` definition.

Reply:
Done